Makefile

```diff
@@ -38,10 +47,18 @@ port-forward-alloy-pprof: kubectl-ctx-kind
 port-forward-hss: kubectl-ctx-kind
 	kubectl port-forward deployment/serverservice ${HSS_PORT}:${HSS_PORT}
 
+## port forward fleetdb port (runs in foreground)
+port-forward-fleetdb: kubectl-ctx-kind
+	kubectl port-forward deployment/fleetdb 8000:8000
```

Comment:
You probably didnt notice the changes I made to this makefile since i did them recently, but could you move the port to the top with the other ports?

Reply:
Also add port forward to the targets port-all-with-lan and port-all, and kill-all-ports